routes/product.js

```diff
@@ -19,3 +21,18 @@ async function createProduct(req, res) {
     res.status(422).json({ message: 'Could not create product', e });
   }
 }
+
+async function getProducts(req, res) {
+  try {
+    const products = await productModel.findAll();
+    console.log("productModel {\n");
+    console.log(productModel);
+    console.log("\n}");
+    res.json({products, message: "success"});
```

Comment:
i usually keep an empty space between opening and closing curly brackets:

`res.json({ products, message: '' });`

also, always use single quotes to stay consistent